test/test_utils.py

```diff
@@ -249,17 +249,36 @@ def _test_sanitize_path(self):
         self.assertEqual(sanitize_path('abc/def...'), 'abc\\def..#')
         self.assertEqual(sanitize_path('abc.../def'), 'abc..#\\def')
         self.assertEqual(sanitize_path('abc.../def...'), 'abc..#\\def..#')
-
-        self.assertEqual(sanitize_path('../abc'), '..\\abc')
-        self.assertEqual(sanitize_path('../../abc'), '..\\..\\abc')
-        self.assertEqual(sanitize_path('./abc'), 'abc')
-        self.assertEqual(sanitize_path('./../abc'), '..\\abc')
-
-        self.assertEqual(sanitize_path('\\abc'), '\\abc')
-        self.assertEqual(sanitize_path('C:abc'), 'C:abc')
-        self.assertEqual(sanitize_path('C:abc\\..\\'), 'C:..')
```

Comment:
> ```py
>        self.assertEqual(sanitize_path('C:abc\\..\\'), 'C:..')
> ```
so we were expecting the wrong result here?

Reply:
yes, since `..` cancels previous path segments, `abc` and `..` should cancel out, and only the root (`c:`) should remain

Reply:
can also be shown using a windows pc:
```py
>>> from nt import _path_normpath as normpath
>>> normpath("C:abc\\..\\")
"C:"
```

Reply:
But how did we manage to write the wrong test?

Reply:
Idk, probably I didn't notice?